quickstart/docker/image/run-with-ziti-cli.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+#
+# Base script for anything that requires the Ziti CLI
+#
+if [ "$#" -ne 1 ]; then
+  echo "Usage: $0 scriptName"
+  exit 0
+fi
+
+if [[ ! -f $1 ]]; then
+  echo "Script $1 not found"
+  echo "Usage: $0 scriptName"
+  exit 0
+fi
+
+# Should we execute?
+initFile="/openziti/access-control.init"
+if [[ -f "${initFile}" ]]; then
+  echo " "
+  echo "*****************************************************"
+  echo " docker-compose init file has been detected"
+  echo " the initialization of the docker-compose environment has already happened"
+  echo " if you wish to allow this volume to be re-initialized, delete the file"
+  echo " located at ${initFile}"
+  echo "*****************************************************"
+  echo " "
+  exit 0
+fi
+
+# give the controller time to ramp up before running if running in docker-compose
+sleep 5
```

Comment:
this should be changed to wait for the controller to start instead of a blanket time

Reply:
actually - i see it waiting down on line 47... i remember now - this is to permit the ziti.env file to exist before continuing. i shoulda made that a loop and wait for the file instead of just '5'...

Reply:
can you see if this works for ya?
```
until $(test -f "${ZITI_HOME}/ziti.env"); do echo "waiting for ziti.env..."; sleep 1; done
```

Reply:
Seems to work just fine.  Pushed a new commit with this change.